velox/docs/functions/spark/json.rst

```diff
@@ -0,0 +1,11 @@
+================
+Json Functions
+================
+
+.. spark:function:: json_object_keys(jsonString) -> array(string)
+
+    Returns all the keys of the outermost JSON object as an array if a valid JSON object is given. If it is any other valid JSON string, an invalid JSON string or an empty string, the function returns null. ::
```

Comment:
Perhaps add examples and tests for `other valid JSON string, an invalid JSON string or an empty string`.

Reply:
Done, thanks